src/main/java/core/basesyntax/figures/Square.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Square extends Figure {
+    private static final String type = "square";
```

Comment:
The 'type' field should not be static. Since 'type' is a characteristic of each instance, it should be a non-static field. Also, consider using a constant for the type name, but ensure that it is not static if it's an instance characteristic.